### ISSUE TITLE
Accessibility Issue: Keyboard users are unable to access the ng-footer-tmp content using the Tab key or other keyboard navigation methods.

### ISSUE THREAD

**Issue by shirishasangam** [ng-select/ng-select#2483]
### Describe the bug

When using ng-select with a custom footer defined using ng-footer-tmp, the content of these templates cannot be accessed via keyboard navigation (e.g., Tab key or other keys).

This is Accessibility issue as the keyboard user cannot access the footer inside select option.

### Steps to Reproduce

1.Execute the example in stackblitz.
2.Open select and try to access Clear button via keyboard.

### Please provide a link to a minimal reproduction of the bug. StackBlitz, CodePen or CodeSanBox

https://stackblitz.com/edit/angular13-recaptcha-v3-hsdw7d?file=src%2Fapp%2Fapp.component.ts

### Expected behaviour

Keyboard user must able to access Clear button inside ng-select

### What version of the library you see this issue?

8.1.1

### What version of Angular are you using?

Angular 13

### Screenshots

![image](https://github.com/user-attachments/assets/865e50d2-9ae4-4b00-8d27-7352374db11c)

### Desktop? Please complete the following information

OS: Windows 11
Browser: Google Chrome, Microsoft Edge

**Comment by pavankjadda** [comment]
Duplicate of #1583
